terps/scott/TI99_4a_terp.c

```diff
@@ -0,0 +1,570 @@
+#include <string.h>
+
+#include "glk.h"
+#include "load_TI99_4a.h"
+#include "scott.h"
+#include "TI99_4a_terp.h"
+
+ActionResultType PerformTI99Line(uint8_t *action_line)

```

Comment:
This should be `static` since it's only used in this file.

In addition, it looks like `action_line` can be `const`.

Reply:
And done.